docs/quickstart/junit_4_quickstart.md

```diff
@@ -0,0 +1,77 @@
+# JUnit 4 Quickstart
+
+It's easy to add Testcontainers to your project - let's walk through a quick example to see how.
+
+Let's imagine we have a simple program that has a dependency on Redis, and we want to add some tests for it.
+ 
+You can see the implementation code and the corresponding test before we've added Testcontainers below:
+
+!!! example "Scenario code"
+    <!--codeinclude-->
+    [Implementation](../examples/src/main/java/quickstart/RedisBackedCache.java) block:RedisBackedCache
+    [Pre-Testcontainers test code](../examples/src/test/java/quickstart/RedisBackedCacheIntTestStep0.java) block:RedisBackedCacheIntTestStep0
+    <!--/codeinclude-->
+
+Notice that the existing test has a problem - it's relying on a local installation of Redis, which is a red flag for test reliability.
+This may work if we were sure that every developer and CI machine had Redis installed, but would fail otherwise.
+We might also have problems if we attempted to run tests in parallel, such as state bleeding between tests, or port clashes.
+
+Let's start from here, and see how to improve the test with Testcontainers:  
+
+## 1. Add Testcontainers as a test-scoped dependency
+
+First, add Testcontainers as a dependency as follows:
+
+```xml tab='Maven'
```

Comment:
General idea:
WDYT about putting `Gradle` first? it saves 4 lines and reduces the length of the docs